shared_task_code/ecs_task_queue_consumer.py

```diff
@@ -0,0 +1,131 @@
+import os
+import aioboto3
+import boto3
+from utils.lambda_decorators import ssm_parameters, async_handler
+from utils.json_serialisation import dumps
+from json import loads
+from datetime import datetime
+import subprocess
+
+
+class ECSTaskQueueConsumer:
+    region = os.environ["REGION"]

```

Comment:
All of this should be done in the __init__ method of the TaskQueueConsumer that this should extend.

There is so much copy paste here that we should avoid.